lang/java/reef-applications/reef-vortex/pom.xml

```diff
@@ -51,48 +51,20 @@ under the License.
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.avro</groupId>
-            <artifactId>avro</artifactId>
+            <groupId>com.esotericsoftware</groupId>
+            <artifactId>kryo-shaded</artifactId>
```

Comment:
I'm not that familiar with licensing, but I assume this is something we can use?